grpc/examples/src/main/scala/sttp/tapir/grpc/examples/GrpcSimpleBooksExample.scala

```diff
@@ -0,0 +1,93 @@
+package sttp.tapir.grpc.examples
+
+import akka.actor.ActorSystem
+import akka.grpc.GrpcClientSettings
+import akka.http.scaladsl.Http
+import cats.implicits._
+import com.typesafe.config.ConfigFactory
+import com.typesafe.scalalogging.StrictLogging
+import sttp.tapir._
+import sttp.tapir.grpc.protobuf._
+import sttp.tapir.grpc.protobuf.model._
+import sttp.tapir.grpc.protobuf.pbdirect._
+import sttp.tapir.server.ServerEndpoint
+import sttp.tapir.server.akkagrpc.AkkaGrpcServerInterpreter
+import sttp.tapir.generic.auto._
+import sttp.tapir.grpc.examples.gen.{SimpleBook => GenSimpleBook, Library => GenLibrary, LibraryClient => GenLibraryClient}
+
+import java.io.ByteArrayOutputStream
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+case class SimpleBook(title: String)
+
+/** Descriptions of endpoints used in the example.
+  */
+object Endpoints {
+  val addBook = endpoint
+    .in("Library" / "AddBook")
+    .in(grpcBody[SimpleBook])
+    .out(grpcBody[SimpleBook])
+
+  val es = List(addBook)
```

Comment:
needs a better name or to be removed

Reply:
added few other endpoints, so renamed it to `endpoints`